### PR TITLE
Update Julia version for GH actions

### PR DESCRIPTION
**Pull request by nefrathenrici**
Updated to 1.8

### CODE REVIEW

File: .github/workflows/CodeCov.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Set up Julia
       uses: julia-actions/setup-julia@latest
       with:
-        version: 1.7.0
+        version: 1.8
```
Comment (nefrathenrici) [inline .github/workflows/CodeCov.yml:22]: @charleskawczynski Should this be more specific, e.g. 1.8.5?

Comment (charleskawczynski) [reply .github/workflows/CodeCov.yml:22]: 1.8 should be fine

**Review by charleskawczynski — APPROVED** [review]

**Comment by charleskawczynski** [comment]
Ah, we need to update the bors toml for these checks!

**Comment by nefrathenrici** [comment]
bors try

**Pull request merged**